diff --git a/Assets/World/C1L1/Waterfall/Shaders/ScrollTextureLayer.shadersubgraph b/Assets/World/C1L1/Waterfall/Shaders/ScrollTextureLayer.shadersubgraph
index 2027fa7..e7c0dab 100644
--- a/Assets/World/C1L1/Waterfall/Shaders/ScrollTextureLayer.shadersubgraph
+++ b/Assets/World/C1L1/Waterfall/Shaders/ScrollTextureLayer.shadersubgraph
@@ -105,20 +105,6 @@
                 "m_SlotId": 1
             }
         },
-        {
-            "m_OutputSlot": {
-                "m_Node": {
-                    "m_Id": "c0a9463a568e4c07a076fa1df0c03ff8"
-                },
-                "m_SlotId": 0
-            },
-            "m_InputSlot": {
-                "m_Node": {
-                    "m_Id": "db2d38f3e9804a118c84a86bcf718670"
-                },
-                "m_SlotId": 1
-            }
-        },
         {
             "m_OutputSlot": {
                 "m_Node": {
